src/middlewared/middlewared/plugins/system.py

```diff
@@ -1067,6 +1068,16 @@ def _get_index(country_columns, column):
 
         raise CallError('Unable to connect to any of the specified UI addresses:\n' + '\n'.join(errors))
 
+    @private
+    async def set_timezone(self):
```

Comment:
Arent't we already doing this in _reload_tinmeservices?

Reply:
Have not thought about that. Maybe we should not do it there? It's a code duplication.

Reply:
If you're going to inspect all use cases, I dont mind either way

Reply:
@william-gr there are no, in TN too (actually that's an issue, who handles timezone change on idle node?)

Reply:
_reload_timeservices does

service.* calls are replicated to standby

Reply:
@william-gr then guess I'll stop playing with fire and trying to look smart :)